pkgs/applications/science/logic/ls4/default.nix

```diff
@@ -0,0 +1,38 @@
+{ zlib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "ls4-${version}";
+
+  version = "1.0";
+
+  buildInputs = [ zlib ];
+
+  buildPhase = ''
+  cd core
+  # These object files were committed into the git repo, perhaps accidentally
+  rm -f aiger.o
+  rm -f aiger.o_32
+  gcc -g -O3 -c aiger.c
```

Comment:
Maybe let's move this to ``preBuild`` and let build call ``make`` would be a better way to package this?

Reply:
Done.